### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by adrienbanse**
@blegat This one's fine? First time I use licenses

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Dionysos
```
Comment (blegat) [inline LICENSE:3]:
```suggestion
Copyright (c) 2024: Contributors to Dionysos: https://github.com/dionysos-dev/Dionysos.jl/contributors
```

Comment (blegat) [reply LICENSE:3]: Otherwise, it seems like a greek god wrote this software

**Comment by adrienbanse** [comment]
Seems like this is the issue with the tests and doc: https://github.com/jump-dev/SDPA.jl/issues/56

**Review by blegat — APPROVED** [review]

**Comment by blegat** [comment]
Let's figure out the SDPA issue on a separate PR

**Pull request merged**